R/esqlabs-env.R

```diff
@@ -31,8 +35,8 @@ esqlabsEnv$heightPerPlotMapping <- 8
 #' getEsqlabsRSetting("packageVersion")
 #' getEsqlabsRSetting("packageName")
 getEsqlabsRSetting <- function(settingName) {
-  if (!(any(names(esqlabsEnv) == settingName))) {
-    stop(messages$errorEsqlabsRSettingNotFound(settingName))
+  if (!(any(names(ospsuiteEnv) == settingName))) {
+    stop(messages$errorPackageSettingNotFound(settingName, ospsuiteEnv))
```

Comment:
Delete the message which is no longer used?

https://github.com/esqLABS/esqlabsR/blob/b17d6195eb7c48c77f3cebfaf6526ebe31f43291/R/messages.R#L16-L18

Reply:
Yepp